h2o-py/h2o/frame.py

```diff
@@ -171,6 +173,10 @@ def get_frame(frame_id):
             return None
         return fr
 
+    def refresh(self):
```

Comment:
It's more of a debug/test function: whenever you call any function on a frame, it may not be immediately executed, but instead stored as a `frame._ex`. At the same time, frame's properties (such as list of columns, their types, frame's shape, etc) will be pre-computed by the client and stored with the frame. This may potentially cause a discrepancy between the client and the server: the client may have computed those properties incorrectly. This function then can be used to fix these issues and reload the frame's properties from the server. It may also be used to eagerize the frame if needed.

This is probably not the best choice of a name for a function, and I'm not yet sure whether this should be public or private...